service/application/src/main/java/com/solace/maas/ep/event/management/agent/cli/CommandLineCommon.java

```diff
@@ -0,0 +1,91 @@
+package com.solace.maas.ep.event.management.agent.cli;
```

Comment:
Pulled out some of the common code to this file that the scan and upload commands can utilize.